core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/diagnostics/DiagnosticsCollectionValidator.java

```diff
@@ -19,16 +21,17 @@ public DiagnosticsCollectionValidator(SupportBundleConfiguration supportBundleCo
         this.supportBundleConfiguration = supportBundleConfiguration;
     }
 
-    public void validate(BaseDiagnosticsCollectionRequest request, Telemetry telemetry, String stackCrn) {
-        validate(request.getDestination(), telemetry, stackCrn, false);
+    public void validate(BaseDiagnosticsCollectionRequest request, Stack stack, Telemetry telemetry, String stackCrn) {
```

Comment:
do we really need the whole stack object?

Reply:
stack is loaded before that anyway, also it's better to use stack instead of other object to not increase the number of parameters